pandas-stubs/_typing.pyi

```diff
@@ -738,7 +738,7 @@ JsonSeriesOrient: TypeAlias = Literal["split", "records", "index", "table"]
 TimestampConvention: TypeAlias = Literal["start", "end", "s", "e"]
 
 CSVEngine: TypeAlias = Literal["c", "python", "pyarrow", "python-fwf"]
-CSVQuoting: TypeAlias = Literal[0, 1, 2, 3]
+CSVQuoting: TypeAlias = Literal[0, 1, 2, 3, 4, 5]
```

Comment:
Can you change this to use the references to `csv.MINIMAL`, `csv.QUOTE_ALL`, etc., You can do tests on the python version in a stub, e.g., 
```python
if sys.version_info >= (3, 12):
    CSVQuoting: TypeAlias = Literal[csv.MINIMAL, #rest of values ]
else:
   CSVQuoting: TypeAlias = Literal[csv.MINIMAL, #rest of values]
```

Reply:
Oh, so maybe we should use `Union[csv.MINIMAL, csv.QUOTE,  # rest of values]` instead.